static-assets/components/cstudio-browse/browse.js

```diff
@@ -197,7 +197,7 @@
     var parsed = JSON.parse(JSON.stringify(obj), function(key, value) {
       if (key === 'children') {
         $.each(value, function(index, elem) {
-          if (elem.numOfChildren === 0) {
+          if (!elem.folder) {
```

Comment:
Would pages with inner children be ignored by this?

Reply:
That's correct. I updated it to show empty folders and also pages with inner children